docs/meshstack.customer-group-sync.md

```diff
@@ -4,3 +4,206 @@ title: meshCustomer Group Synchronisation
 ---
 
 meshStack supports importing users and groups into meshStack from a source that supports the LDAP protocol, for example, Microsoft Active Directory. The [`Simple Paged Results Control`](https://www.ietf.org/rfc/rfc2696.txt) is used to do a paginated LDAP query to fetch the users and groups. Because of this, it is required that the LDAP server supports the `Simple Paged Results Control`. Other than importing users and groups, meshStack also has the capability to assign a [customer role](./meshcloud.customer.md#assign-meshcustomer-roles) to the imported group. The entities read via LDAP are imported to meshStack via the [meshObject import API](./meshstack.api.md#meshobject-api).
+
+## Configuration Reference
+
+This section describes how to configure the LDAP group synchronization in meshStack.
+For easier reference the following sections break down the configuration model in multiple parts. The union of these
+defines the full configuration model.
+
+The basic parameters required to configure the connection to LDAP are as follows.
+
+<!--snippet:mesh.identityconnector.ldap.core#type-->
+
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Dhall Type-->
+```dhall
+let ConnectionConfiguration =
+    {-
+      url: The LDAP connection URL which would be of the form ldap://example.com:389
+      base:
+          The base from which searches should be performed, which would be of the form "dc=meshcloud,dc=io"
+      username: The username of the user who can connect to the LDAP source to perform searches
```

Comment:
this is a technical user right not a personalized one. Think we should mention that somewhere

Reply:
Not sure if ActiveDirectory (or other LDAP sources) have the concept of a technical user. I could say technical user here but not sure if that would clarify or confuse.

Reply:
Will say `technical user`. I suppose they would understand.